test/jdk/java/awt/color/CheckDefaultProperties.java

```diff
@@ -34,8 +34,9 @@
 
 /**
  * @test
- * @bug 8256321
- * @summary Verifies profile properties are the same before/after activation
+ * @bug 8256321 8359380
+ * @summary Verifies built-in profile properties are the same before and after
+ *          activation and in copies of built-in profiles
```

Comment:
The test compared data from the lightweight object with the actual data used after the profile is fully loaded. Since we now always use the lightweight object, I had to create a copy of the standard profile to make sure the actual profile data is used.